docs/7.x/pack.md

```diff
@@ -1117,3 +1117,21 @@ When building a Cluster Image, `tele build` will discover `custom-planet:1.0.0`
 Docker image and vendor it along with other dependencies. During the Cluster
 installation all nodes with the role `worker` will use the custom "planet" Docker
 image instead of the default one.
+
+### Kubenetes Admission Controllers
+
+Optional Kubernetes admission controllers can be enabled when using a custom `planet`
+image. Create an environment file and specify a list of admission controllers to
```

Comment:
This should mention that this is in addition to a set of predefined addmission controllers.

Reply:
Good call. I'll include a sentence about the admissions controllers that are enabled by default.